.github/workflows/tests.yaml

```diff
@@ -0,0 +1,30 @@
+name: Tests
+
+on:
+  push:
+  pull_request:
+
+permissions:
+  contents: read
```

Comment:
I am not knowledgeable on GitHub Actions, but wondering if we need to have also `pull-requests: read`. Could this be otherwise ignoring tests introduced in PRs?

Reply:
> Could this be otherwise ignoring tests introduced in PRs?
This is a PR and it introduce tests and the test runs so I believe it's fine. Do note though that I'm neither an expert on GH actions and that I've followed Paulo's example here: https://github.com/rancher/cis-operator/pull/267

Reply:
`on` triggers and `permissions` are orthogonal to each other.

Reply:
More on permissions https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idpermissions

Reply:
`contents: read` is enough in this case.